lms/djangoapps/teams/tests/test_views.py

```diff
@@ -508,6 +524,12 @@ def test_expand_public_user(self):
     )
     @ddt.unpack
     def test_text_search(self, text_search, expected_team_names):
+        # clear out the teams search index before reindexing
+        CourseTeamIndexer.engine().destroy()
+
+        for team in self.test_team_name_id_map.values():
+            CourseTeamIndexer.index(team)
+
```

Comment:
Just to clarify, this needs to be added to force indexing because it's not automatically happening due to signals, right?